src/plugins/quake-pro/quake-pro.replacements

```diff
@@ -0,0 +1,43 @@
+# Replacements for Quake Pro
+
+*plugin* `quake-pro`
+
+
+## Increase the FOV of the camera
+
+*Description* `Multiply getPerspectiveDistortion() to increase the viewport FOV.`
+
+*Find* => `from`
+```js
+{
+  let $e = this.graphSettings.config.perspectiveDistortion;
+  return __return__
+}
+```
+
+*Replace* `from` with
+```js
+{
+  let $e = this.graphSettings.config.perspectiveDistortion;
+  return __return__ * (DSM.quakePro?.magnification ?? 1);
```

Comment:
The function currently looks like
```
        getPerspectiveDistortion() {
            let e = this.graphSettings.config.perspectiveDistortion;
            return isNaN(e) ? 1 : e < 1e-4 ? 0 : clamp(e, 0, xk)
        }
```
The replacement is token-based, so it will only multiply the clamp:
```
        getPerspectiveDistortion() {
            let e = this.graphSettings.config.perspectiveDistortion;
            return isNaN(e) ? 1 : e < 1e-4 ? 0 : clamp(e, 0, xk) * (DSM.quakePro?.magnification ?? 1);
        }
```

Is this what you expect? Maybe the 1 should be multiplied by the magnification, which would be putting parentheses around the return.
```suggestion
  return (__return__) * (DSM.quakePro?.magnification ?? 1);
```